NEWS.md

```diff
@@ -1,10 +1,72 @@
 # Development
 
-## New features
+## New features and changes
+
+- The new `.include` argument of `config_log()`, `run_log()`, and
+  `summary_log()` limits the result to the specified run names or tags.  (#484,
+  #526)
+
+- Models can now be starred.  See `add_star()` and `remove_star()`.  (#487)
+
+- New convenience functions `get_omega()`, `get_sigma()`, and `get_theta()`
+  return labeled values, with the `OMEGA` and `SIGMA` values expanded to a full
+  matrix.  (#515)
+
+- `param_estimates` gained an `.alpha` argument that's useful for identifying
+  the ETAs flagged by the `eta_pval_significant` heuristic.  (#497)
+
+- `nm_join()` now aborts if the join column has duplicate values, suggesting to
+  the caller that the table format probably needs to be widened to prevent
+  NONMEM from truncating the values.  (#533)
+
+- `nm_file()` and friends now detect duplicate column names and make them
+  unique.  (#530, #539)
+
+- If the caller specifies a `threads` value above one in `.bbi_args` but does
+  _not_ specify `parallel`, `parallel = TRUE` is added so that the `threads`
+  value is in effect.  (#514)
+
+- New functions `check_nonmem_finished()` and `wait_for_nonmem()` enable
+  checking and waiting on models submitted to the grid.  (#480)
+
+- Added `test_threads()` for benchmarking simulation run times with various
+  threads values.  (#473, #489, #519)
 
-- Added `test_threads()` for benchmarking simulation run times with various threads values.
 - Added `check_run_times()` for checking the estimation times of model runs.
-- Added `delete_models()` for removing all model files associated with specified model tags.
+  (#473, #489, #511)
+
+- Added `delete_models()` for removing all model files associated with
+  specified model tags.  (#473)
+
+- bbi encodes "unspecified" and `NA` values as `-999999999`.  bbr now maps all
+  occurrences of this value to `NA` when creating a model summary object.
+  (#524)
+
+- When bbi v3.2.0 or later is available, `model_summaries()` now uses bbi's
+  concurrency rather than calling `model_summary()` on each model, leading to a
+  speed up when many models are passed.  (#527)
+
+- `model_diff()` learned to print a message rather than call
+  `diffobj::diffFile()` when there are no changes to avoid confusingly
+  displaying the entire file.  (#522)
+
+- `use_bbi` now creates leading directories if needed.  (#499)
+
+- `new_model` now ignores the extension of the supplied path.  (#510)
+
```

Comment:
How do you feel about adding `read_model()` to this, and also adding "valid" before extensions, given that not all extensions are technically ignored (just `.ctl` and `.mod` from what I remember. 

In my "fake news" branch I had (though an example might not be that important):
`new_model() and read_model() will now ignore valid extensions (new_model("1.ctl") works when new_model("1") was previously required). (#510)`

Reply:
Ahh I take back the "valid" part. Looked through that PR and it seems all extensions are allowed. Though we only have tests for `.mod` and `.ctl`, which is interesting given that technically `.blah` would seemingly work

Reply:
> Though we only have tests for .mod and .ctl

There's really not much point in doing comprehensive testing of `fs::path_ext_remove`.

> How do you feel about adding read_model() to this

I don't think that'd be accurate, but I'll confirm that I'm not mistaken.

Reply:
> I don't think that'd be accurate, but I'll confirm that I'm not mistaken.

Yeah its not accurate. I recall talking with RyanM about doing that on a zoom call (and swore I saw it being added), but guess it didnt make it through